libphal/phal_exception.C

```diff
@@ -0,0 +1,24 @@
+#include "log.H"
+#include "phal_exception.H"
+
+namespace openpower::phal
+{
+namespace exception
+{
+
+using namespace openpower::phal::logging;
+
+SbeError::~SbeError()
+{
+	if (!fileName.empty()) {
+		if (remove(fileName.c_str())) {
+			log(level::ERROR, "File(%s) remove failed(errnno:%d)",
+			    fileName.c_str(), errno);
+		} else {
+			// set file descriptor value to default
```

Comment:
isn't this valid for both if and else, setting fd=-1, actually there is no need also as we are in the destructor

Reply:
whole else can be removed, who cares what fd has after destruction

Reply:
good point . added initially for readbilty , as you said sinnce wer are i destructor , not required.